src/store/Modules/NewProject.ts

```diff
@@ -0,0 +1,140 @@
+import { Module, VuexModule, Mutation } from "vuex-module-decorators"
+import store from "../index"
+const storeValue = localStorage.getItem("vuex")
+const newProjectInStore = storeValue ? JSON.parse(storeValue).NewProject : false
+@Module({
+  name: "NewProject",
+  store: store,
+  dynamic: true,
+  preserveState: newProjectInStore,
+})
+export default class NewProject extends VuexModule {
```

Comment:
this class shoule be called Projects. 
this file should be called Projects.

Reply:
Class name and file name modified